src/reference/environments/example_envs.rst

```diff
@@ -0,0 +1,76 @@
+Example Environments
+====================
+
+A set of possible installations of Cylc:
+
+.. csv-table::
+   :header-rows:1
```

Comment:
This isn't formatting right for me locally

![image](https://user-images.githubusercontent.com/61982285/132215322-b84b406d-0375-4592-8adc-36bdeb1240a0.png)

Reply:
apparently the problem was:
```diff
-   :header-rows:1
+   :header-rows: 1